src/main/scala/exercises/06_Collections.scala

```diff
@@ -50,9 +48,9 @@ object Collections {
    *  - by using a foldLeft function
    *  - ... etc
    */
-  def lastElementInList[T](l: List[T]): T = {
-    error("fix me")
-  }
+  def lastElementInList[T](l: List[T]): T = l.last
+  // not intuitive: if getting the first element can be done by l.head, then I would 
+  // expect to get the last element by calling l.tail
```

Comment:
I didn't think `tail` was unintuitive, but that's probably because I know other functional programming languages use `tail`. Alternatively, maybe `first` and `rest` is more intuitive to some people.